common/laws/00_health_system.txt

```diff
@@ -148,7 +148,7 @@ law_public_health_insurance = {
 	}
 	
 	modifier = {
-		country_bureaucracy_investment_cost_factor_mult = 0.20
+		country_institution_cost_institution_health_system_mult = 1.5
```

Comment:
Hear me out: institution level related, institution cost modifier xd (for polynomial increase in costs instead of linear).

Reply:
Tried that with the Citizenship Institution way back when and it appeared to be nonfunctional. Although it was a different modifier, I don't think modifiers that affect what is being modified work properly.